src/test/java/com/salaboy/conferences/agenda/rest/ContractVerifierBase.java

```diff
@@ -1,24 +1,31 @@
 package com.salaboy.conferences.agenda.rest;
 
+import com.salaboy.conferences.agenda.rest.repository.AgendaItemRepository;
 import io.restassured.RestAssured;
 import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.TestInstance;
 import org.mockito.MockitoAnnotations;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.boot.web.server.LocalServerPort;
 import org.springframework.test.context.ActiveProfiles;
 
 
-@SpringBootTest( webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
-        properties = "server.port=0")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
+        properties = {"server.port=0"})
 @ActiveProfiles("dev")
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
 public abstract class ContractVerifierBase {
 
     @LocalServerPort
     int port;
 
+    @MockBean
+    AgendaItemRepository agendaItemRepository;
```

Comment:
When the contract verifier test runs, shows it:
```bash
2022-04-28 22:33:10.046  INFO 1936183 --- [           main] c.s.c.agenda.rest.ContractVerifierBase   : Set up ContractVerifierBase
2022-04-28 22:33:11.116  INFO 1936183 --- [or-http-epoll-3] c.s.c.a.r.controller.AgendaController    : > REST ENDPOINT INVOKED for Creating a new Agenda Item
2022-04-28 22:33:11.117  INFO 1936183 --- [or-http-epoll-3] c.s.c.a.rest.service.AgendaItemService   : > New Agenda Item Received: AgendaItem{id='null', proposalId='null', title='UERSRHNAVXBFXYTWVCNM', author='ICVZKCHYQBAXXKAOHPTE', day='EUBPMKSGWTYUETNQTPAS', time='MWYPFRFYQUKHUJYPNJFD'}
2022-04-28 22:33:11.388  INFO 1936183 --- [or-http-epoll-3] i.j.internal.reporters.LoggingReporter   : Span reported: 623e0f45d671b5a9:623e0f45d671b5a9:0:1 - newAgendaItem
2022-04-28 22:33:11.412 ERROR 1936183 --- [or-http-epoll-3] a.w.r.e.AbstractErrorWebExceptionHandler : [ccc2869f-1]  500 Server Error for HTTP POST "/"

org.springframework.data.redis.RedisConnectionFailureException: Unable to connect to Redis; nested exception is io.lettuce.core.RedisConnectionException: Unable to connect to localhost:6370
```

Solution: 
- Add a mock for AgendaItemRepository

If you know a better way to solve this issue, tell me please :rocket:

Reply:
for testing this should be ok..